### PR TITLE
Delete My Application 1220

### LINKED ISSUE #1220: Implement My Profile - My Applications delete

### PR DESCRIPTION
**Pull request by na3d**
Closes #1220

screen shot 2017-03-07 at 5 12 55 pm

### CODE REVIEW

**Comment by na3d** [comment]
screen shot 2017-03-08 at 2 27 20 pm

**Comment by na3d** [comment]
screen shot 2017-03-08 at 2 53 13 pm

File: src/profile/integrations/components/MyApplication.js [hunk 1]
```diff
@@ -1,44 +1,102 @@
...
+        this.deleteAction();
+      }
+    };
+  }
+
+  renderActions() {
+    const elements = [
+      { _action: this.editApp, name: 'Edit' },
+      { _action: this.deleteApp, name: 'Delete' },
+    ].map(element => ({ ...element, action: this.passAction(element._action) }));
```
Comment (eatonphil) [inline src/profile/integrations/components/MyApplication.js]: Huh? This looks funky. Can we just drop the map part?

File: src/profile/integrations/components/MyApplication.js [hunk 2]
```diff
@@ -1,44 +1,102 @@
...
+
+      if (action === 'Delete') {
+        this.deleteAction();
+      }
+    };
+  }
+
+  renderActions() {
+    const elements = [
+      { name: 'Edit', action: this.passAction('Edit') },
```
Comment (eatonphil) [inline src/profile/integrations/components/MyApplication.js]: Sorry, can these just be `action: () => this.editAction()` instead? And drop the passAction?

**Pull request merged**